src/Controller/Component/SecurityComponent.php

```diff
@@ -486,9 +486,9 @@ public function generateToken(ServerRequest $request): ServerRequest
             'unlockedFields' => $this->_config['unlockedFields'],
         ];
 
-        $request->getSession()->write('_Token', $token);
+        $request->getSession()->write('_formToken', $token);
```

Comment:
I couldn't find anywhere where this value is read back from session. So is this statement required?

Reply:
Probably not anymore. This may be a relic of the `allowedControllers` and `allowedActions` features.

Reply:
Okay, I have removed it.